### DISCUSSION TITLE
exception of distutils when importing PyTorch_lightning

### DISCUSSION THREAD

**Discussion by lynnfyffe** [Lightning-AI/pytorch-lightning#15575] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
import pytorch_lightning as pl

"Exception has occurred: AttributeError
module 'distutils' has no attribute 'version'"

**Comment by Borda** [comment]
Can you share what PL and `distutils` you are using?

**Reply by lynnfyffe** [reply]
pytorch-lightning         1.7.7 
distutils not found
python                          3.10.6

**Comment by akihironitta** [comment]
Please refer to #12324.
